src/components/product/ProductCard.tsx

```diff
@@ -1,9 +1,18 @@
 'use client';
 import React, { useState } from 'react';
+import dynamic from 'next/dynamic';
+import Image from 'next/image';
 import { ColorDots } from '@/components/product/ColorDots';
 import { StockStatus } from '@/components/product/StockStatus';
-import { PlaceOrderModal } from '@/components/modals/PlaceOrderModal';
-import Image from 'next/image';
+// import { PlaceOrderModal } from '@/components/modals/PlaceOrderModal';
```

Comment:
I've noticed a tendency to leave commented-out code or unused components in the codebase. Could you please remove unnecessary elements.

Reply:
great catch! it wasn't intentional, but my omission. will fix asap.